gp-limit-submissions/gpls-disable-rule-for-user-role.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Gravity Perks // GP Limit Submissions // Disable Rule for a Specific User Role
+ * http://gravitywiz.com/documentaiton/gravity-forms-limit-submissions/
+ */
+// Change '123' to the ID of the Form.
+add_filter( 'gpls_rule_groups_123', function( $rule_groups, $form_id ) {
+	$user          = wp_get_current_user();
+	$role_to_check = 'administrator'; //Change to the slug of the role, i.e. administrator, editor, author, contributor, subscriber.
```

Comment:
Let's move this to the first line. Then, update the comment to be above the variable and change the word "Change" to "Update" to follow the same style we use with other snippets.